manager-bundle/src/Resources/skeleton/app/config.yml

```diff
@@ -98,3 +98,4 @@ fos_http_cache:
         enabled: true
     tags:
         enabled: true
+        max_header_value_length: 2048
```

Comment:
The documentation says:

> 4096 bytes is generally a good choice because it seems like most webservers have a maximum value of 4 KB configured.

https://github.com/FriendsOfSymfony/FOSHttpCacheBundle/blob/master/Resources/doc/reference/configuration/tags.rst#max_header_value_length

Why are we using `2048` then?

Reply:
Just to be safe, but we might as well use 4096. The thing is, it doesn't really help anything at all anyway. E.g. nginx treats multiple headers the same as one big header so even if we split it into 5 headers, as long as they're named the same, nginx will take the total of the headers to determine the length.

Reply:
That sucks big time. All the effort for nothing. 😢

Then we should not merge this PR, especially not the part which removes the header in dev mode. What about extending the web toolbar so it shows the headers and removes them from the response?

Reply:
We must merge it, for sure. Here are the reasons:

* The effort wasn't for nothing. Just because nginx does not support it (which also might have been a local config issue), does not mean that it's the same for other servers.
* Removing in `app_dev.php` is absolute nonsense to me. My opinion is that if it fails, it's **a good thing**. I want consistency and it's very weird that it does not fail in `dev` mode but it does in `prod` when using e.g. Varnish.

Reply:
I agree 50% to the first reason and 100% to the second one. 😄

Reply:
> which also might have been a local config issue

I have read some Nginx docs as well as the source code and it does not seem to be configurable. Might have missed it though.

Reply:
Yeah well, now that we have an `array_unique()` on the tags you really have to have loads of tags and then increasing the header buffer size of the server is probably the way to go 😄 Should not be a big deal if you can afford having Varnish in front of your server 😄